src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.h

```diff
@@ -7,6 +7,16 @@
 
 #include <cstdint>
 
+// InitializeInstrumentation needs to be called once after this library is injected into the target
+// process. It sets up the communication to OrbitService.
+extern "C" void InitializeInstrumentation();
+
+// We'll spawn six threads when injecting this library. This happens immediatly after the call to
```

Comment:
I think we should mention why these threads are spawned (communication/gRPC/etc.), either here or in `GetNewOrbitThreads` where they are listed.

Reply:
done